app/graph/types/user_type.rb

```diff
@@ -17,4 +17,18 @@ class UserType < DefaultObject
   def source
     Source.find(object.source_id)
   end
+
+  def name
+    meedan_user? ? 'Meedan' : object.name
+  end
+
+  def profile_image
+    meedan_user? ? 'http://localhost:3000/images/checklogo.png' : object.profile_image
```

Comment:
This hard-coded `http://localhost:3000/` should be replaced by `CheckConfig.get('checkdesk_base_url')`.